canonical-web-and-design/conjure-up.io
# Multistage build to reduce docker image sizes

## Done

Implemented multistage build to reduce docker image sizes using reference from ubuntu.com

## QA

- Changes should not affect site
- Check image size: `docker build --tag myimage --build-arg BUILD_ID=myfakeid . && docker images` check myimage size
- Revision ID is correctly added in the headers
- `docker run -ti -p 8111:80 myimage` and check the website is running correctly
- Go to your browser to see this is running correctly.
- `curl -I http://localhost:8111` to see if headers display "myfakeid" (or inspect developer tools -> network)

## Issue / Card

- Multistage builds issue (WUB Fix #2142)
- Standardize build-ids (WUB Fix #2096)
